ruby/README.md

```diff
@@ -1,41 +1,25 @@
-# AWS SDK for Ruby - Code Examples
-[![Ruby Style Guide](https://img.shields.io/badge/code_style-rubocop-brightgreen.svg)](https://github.com/rubocop/rubocop)
-
+# AWS SDK for Ruby code examples
 ## Overview
+The code examples in this topic show you how to use the AWS SDK for Ruby with AWS. 
```

Comment:
Keep the SDK version here: "AWS SDK for Ruby Version 3" for clarity.

Reply:
I removed it because the difference is the structuring of the libraries (and as a result, how you import them) and the code itself is backwards compatible, so I don't want to confuse people. I can add it back if you feel strongly though.